utils/useful.py

```diff
@@ -236,80 +266,89 @@ def created_at(self) -> datetime.datetime:
     def remove_message(self, message_id: int) -> Optional[discord.Message]:
         return self.sent_messages.pop(message_id, None)
 
-    async def maybe_reply(self, content: str = None, mention_author: bool = False, **kwargs: Any) -> discord.Message:
+    async def maybe_reply(self, content: Optional[str] = None, mention_author: bool = False,
+                          **kwargs: Any) -> discord.Message:
         """Replies if there is a message in between the command invoker and the bot's message."""
         await asyncio.sleep(0.05)
         with contextlib.suppress(discord.HTTPException):
             if ref := self.message.reference:
                 author = ref.cached_message.author
-                mention_author = mention_author or author in self.message.mentions and author.id not in self.message.raw_mentions
+                mention_author = mention_author or author in self.message.mentions \
+                    and author.id not in self.message.raw_mentions
```

Comment:
if you're gonna do this, might as well just do it the proper way jesus christ

Reply:
I think this is resolved